src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
+@import './header';
+@import './main';
+@import './variables';
+@import './card';
```

Comment:
The order of imports should be as follows: variables, then other SCSS files. So, you should move the import of './variables' to the first place.